### PR TITLE
fix: remove prettier path

### LINKED ISSUE #3: Doesn't work when Prettier is only installed globally
I have `prettier` and `prettier-eslint-cli` installed globally because I want to use it in any JavaScript project with vim.

When I format a file, I get the text of the file replaced with this:

```
prettier-eslint-cli error: There was a problem trying to format the stdin text Error: Cannot find module '../../../../../../path/to/current/project/node_modules/prettier'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at prettify (/usr/local/lib/node_modules/prettier-eslint-cli/node_modules/prettier-eslint/dist/index.js:79:16)
    at format (/usr/local/lib/node_modules/prettier-eslint-cli/node_modules/prettier-eslint/dist/index.js:66:18)
    at _callee2$ (/usr/local/lib/node_modules/prettier-eslint-cli/dist/format-files.js:77:54)
    at tryCatch (/usr/local/lib/node_modules/prettier-eslint-cli/node_modules/regenerator-runtime/runtime.js:64:40)
    at Generator.invoke [as _invoke] (/usr/local/lib/node_modules/prettier-eslint-cli/node_modules/regenerator-runtime/runtime.js:355:22)
    at Generator.prototype.(anonymous function) [as next] (/usr/local/lib/node_modules/prettier-eslint-cli/node_modules/regenerator-runtime/runtime.js:116:21)
```

But when I install `prettier` locally in the project, it works. I feel like it should work either way.

### PR DESCRIPTION
**Pull request by aharris88**
Closes #3

### CODE REVIEW

File: src/parser.js [hunk 1]
```diff
@@ -14,10 +14,6 @@ const parser = yargs
       default: getPathInHostNodeModules('eslint'),
       describe: 'The path to the eslint module to use',
     },
-    prettierPath: {
```
Comment (kentcdodds) [inline src/parser.js]: Could we do the same to the eslint one too?

Comment (kentcdodds) [inline src/parser.js]: Oh, and don't remove the whole option. Just the default property. And you'll need to update snapshots.

**Comment by kentcdodds** [comment]
Learn more about Jest here: http://kcd.im/egghead-jest

File: src/parser.js (hunk 1 above)
Comment (aharris88) [inline src/parser.js]: I tried removing the eslint one and I got this error:

```
prettier-eslint error: Unable to find config Error: Failed to load plugin json-lite: Cannot find module 'eslint-plugin-json-lite'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.load (/Users/adamharris/Development/github/aharris88/prettier-eslint-cli/node_modules/eslint/lib/config/plugins.js:129:26)
    at Array.forEach (native)
    at Object.loadAll (/Users/adamharris/Development/github/aharris88/prettier-eslint-cli/node_modules/eslint/lib/config/plugins.js:151:21)
    at Object.load (/Users/adamharris/Development/github/aharris88/prettier-eslint-cli/node_modules/eslint/lib/config/config-file.js:504:21)
    at loadConfig (/Users/adamharris/Development/github/aharris88/prettier-eslint-cli/node_modules/eslint/lib/config.js:63:33)
    at getLocalConfig (/Users/adamharris/Development/github/aharris88/prettier-eslint-cli/node_modules/eslint/lib/config.js:130:29)
prettier-eslint-cli error: There was a problem trying to format the stdin text Error: Failed to load plugin json-lite: Cannot find module 'eslint-plugin-json-lite'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.load (/Users/adamharris/Development/github/aharris88/prettier-eslint-cli/node_modules/eslint/lib/config/plugins.js:129:26)
    at Array.forEach (native)
    at Object.loadAll (/Users/adamharris/Development/github/aharris88/prettier-eslint-cli/node_modules/eslint/lib/config/plugins.js:151:21)
    at Object.load (/Users/adamharris/Development/github/aharris88/prettier-eslint-cli/node_modules/eslint/lib/config/config-file.js:504:21)
    at loadConfig (/Users/adamharris/Development/github/aharris88/prettier-eslint-cli/node_modules/eslint/lib/config.js:63:33)
    at getLocalConfig (/Users/adamharris/Development/github/aharris88/prettier-eslint-cli/node_modules/eslint/lib/config.js:130:29)
```

**Comment by aharris88** [comment]
I use Jest at work. I just didn't realize any tests were failing.

File: src/parser.js (hunk 1 above)
Comment (aharris88) [inline src/parser.js]: Ok, now I just removed the default option.

**Comment by aharris88** [comment]
Looks like all tests are passing, including snapshots.

**Comment by koolay** [comment]
release?

**Comment by kentcdodds** [comment]
All pull requests are released automatically when merged. This was [released](https://travis-ci.org/prettier/prettier-eslint-cli/builds/194345543) way back as v1.0.3

**Pull request merged**